lxd/main_activateifneeded.go

```diff
@@ -113,10 +112,22 @@ func (c *cmdActivateifneeded) Run(cmd *cobra.Command, args []string) error {
 
 	var instances []db.Instance
 	err = d.cluster.Transaction(func(tx *db.ClusterTx) error {
```

Comment:
All this could probably just be `instances, err := instance.LoadNodeAll(s, instancetype.Any)` which should give all of them pre-loaded in a more optimized way than the code below.